src/sentry/static/sentry/app/components/pickProjectToContinue.tsx

```diff
@@ -31,7 +31,7 @@ function PickProjectToContinue({
 }: Props) {
   let navigating = false;
 
-  const path = `${nextPath}?project=`;
+  const path = nextPath.includes('?') ? `${nextPath}&project=` : `${nextPath}?project=`;
```

Comment:
Can there be other question marks in the path other than the one you mean?

Reply:
not at the moment, but in the future, if necessary, we can update this logic 😉